leveller.js

```diff
@@ -61,16 +61,24 @@
     var diff = targetHeight - currentHeight;
     if (diff === 0) return;
 
+    var isHeightProperty = (this.options.cssProperty.indexOf('eight') > 0);
```

Comment:
Is the name of the game here to see if the string `height` is in `cssProperty`? If so, might I recommend:

`this.options.cssProperty.indexOf('height') !== -1`

If `height` needs to be at the beginning of the string:

`this.options.cssProperty.indexOf('height') === 0`

Because of the way it's written I can't tell which is your intent :)